recipes/sidekiq.rb

```diff
@@ -20,15 +20,14 @@ def cook
 
     def add_initializer
       @template.initializer 'sidekiq.rb', <<~CODE
-        redis_host = Redis.current.client.host
-        redis_port = Redis.current.client.port
+        redis_url = Redis.current.id
```

Comment:
Why do we use the Redis ID as the redis URL? If we want to use a remote Redis server, then it's not going to work. Could you better explain this change @artero ?

Reply:
I'm not sure but, as I understand now the id, is the redis url:

Now the Redis objects don't have the url or host methods. 

I think the configuration in the PR can work with the next cases:

```ruby
Redis.new.id
=> "redis://127.0.0.1:6379/0"

Redis.new(path: "/tmp/redis.sock").id
=> "redis:///tmp/redis.sock/0"

Redis.new(url: "redis://:p4ssw0rd@10.0.1.1:6380/15").id
=> "redis://10.0.1.1:6380/15"
```

Do you think any other configuration that can fail?

Reply:
It seems to be working fine. Thank you @artero  👍